README.md

```diff
@@ -53,14 +55,22 @@ This Integration Daemon Postgres connector supports the following types of datab
 * Foreign Keys 
 
 # Starting the Egeria Platform
-Below is an example of the java commandline used to launch the Egeria platform that will host the Integration Daemon that is in turn hosting the 
-Postgres connector instance we are about to configure.
+There options when it comes to running Egeria, if using the egeria docker image see https://github.com/odpi/egeria/tree/master/open-metadata-resources/open-metadata-deployment/docker/egeria 'Extending the image' which refers to how to add additional connectors.
```

Comment:
There options
 - typo
 but not a concern.. I'll write up my example for you in an additional PR

Reply:
Further examples will be added when I do https://github.com/odpi/egeria-database-connectors/issues/81
Updated link (after charts move)